src/bindist/fdppauto.bat

```diff
@@ -10,5 +10,5 @@ echo Welcome to dosemu2!
 system -s DOSEMU_VERSION
 echo     Build %DOSEMU_VERSION%
 call exechlp.bat -ep
-if exist %DOSEMUDRV%:\dosemu\dosrc.d\sugfdusr.bat call %DOSEMUDRV%:\dosemu\dosrc.d\sugfdusr.bat
+if exist %DOSEMUDRV%:\dosemu\dosrc.d for %%b in (%DOSEMUDRV%:\dosemu\dosrc.d\*.bat) do call %%b
```

Comment:
Why do you need double percent here?

Reply:
Other DOSs require this apparently when used in batch files. You made something more flexible? Does it also support more than one character?

Reply:
Your fancy comcom32 batch syntax flexibility scares Kate's syntax highlighting :D
![Screenshot_20200820_222017](https://user-images.githubusercontent.com/877112/90821680-945a3b00-e333-11ea-90d3-8faf538b262b.png)

Adding a `%` makes it:
![Screenshot_20200820_222205](https://user-images.githubusercontent.com/877112/90821740-b0f67300-e333-11ea-9ff8-80757b114824.png)

Reply:
https://www.poftut.com/loops-windows-ms-dos-command-propmpt/
In the examples here I do not see 2 percents.
As for the syntax - you can omit % completely
when declaring the iterator, and then the syntax
highlighting will work.